### PR TITLE
Add 1290_Convert_Binary_Number_in_a_Linked_List_to_Integer.py and 1342_Number_of_Steps_to_Reduce_a_Number_to_Zero.py 

### PR DESCRIPTION
**Pull request by srivastavaayu**
Added Convert Binary Number in a Linked List to Integer (problem code - 1290) and Number of Steps to Reduce a Number to Zero (problem code - 1342)

### CODE REVIEW

**Comment by qiyuangong** [comment]
Hi @srivastavaayu 
Thank you for contribution! 👍 

Pls add one line description for your solution, e.g.,

```
| 12 | [Integer to Roman](https://leetcode.com/problems/integer-to-roman/) | [Python](https://github.com/qiyuangong/leetcode/blob/master/python/012_Integer_to_Roman.py) | [Background knowledge](http://www.rapidtables.com/convert/number/how-number-to-roman-numerals.htm) Just like 10-digit number, divide and minus |
```

File: python/1290_Convert_Binary_Number_in_a_Linked_List_to_Integer.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    def getDecimalValue(self, head: ListNode) -> int:
+        ll=[]
+        ll.append(head.val)
+        while(head.next!=None):
+            head=head.next
+            ll.append(head.val)
+        ans=0
+        po=0
+        for i in range(len(ll)-1,-1,-1):
+            ans+=((2**po)*ll[i])
```
Comment (qiyuangong) [inline python/1290_Convert_Binary_Number_in_a_Linked_List_to_Integer.py]: Add ll[i] >= 1 check will cut unnecessary computation. :)
```
            if ll[i]:
                ans+=((2**po)*ll[i])
```

File: python/1342_Number_of_Steps_to_Reduce_a_Number_to_Zero.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Output: 12
+
+Constraints:
+0 <= num <= 10^6
+'''
+
+class Solution:
+    def numberOfSteps (self, num: int) -> int:
+        steps=0
+        while(num>0):
```
Comment (qiyuangong) [inline python/1342_Number_of_Steps_to_Reduce_a_Number_to_Zero.py:36]: Nice solution. 

BTW, in https://leetcode.com/problems/number-of-steps-to-reduce-a-number-to-zero/discuss/502809/just-count-number-of-0-and-1-in-binary, there is an interesting solution based on binary encoding.

**Comment by qiyuangong** [comment]
Both solutions work well on leetcode.Good job.

Pls refine code style based on Pylint (you can install PEP8 or pylint plugin in you IDE), and add necessary comments to increase readability. :)

**Comment by srivastavaayu** [comment]
@qiyuangong I've made necessary changes as suggested by you. Please review the changes. And don't forget to change the links to the solutions in the README after pulling the commits.

**Comment by qiyuangong** [comment]
> @qiyuangong I've made necessary changes as suggested by you. Please review the changes. And don't forget to change the links to the solutions in the README after pulling the commits.

OK. I will 👍

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**